app/helpers/offender_helper.rb

```diff
@@ -30,4 +30,18 @@ def event_type(event)
       type + 'allocated'
     end
   end
+
+  def approaching_handover_without_com?(offender)
+    return false unless offender.sentenced?
```

Comment:
I think this method might be nicer as a 'traditional if statement'. If I'm reading it correctly, it can return false even when com_name is not present. (if there's no email address to talk to) which seems a little counter-intuitive given the name of the method. 
I also wonder if it would be better if it returned the positive case rather than the negative one? (unsure)

Reply:
Maybe this should be split into 2 for readability. approaching_handover_without_com? sounds like a question, as does offender.has_ldu_email? Then the checks can be more explicit i.e. if approaching_handover_without_com? && offender.ldu_email_missing?